### PR TITLE
support comma-separated tag values

### PR DESCRIPTION
**Pull request by Fuud**
**Steps to reproduce:**
1) register metric with comma in tag value
```
		registry.gauge(
			"metricWithCommaInTag",
			Arrays.asList(Tag.of("tagName", "tagValue, ")),
			this,
			value -> 1.0
		);
```
2) try to add it on Admin Server UI
image
3) check network in browser:
```{"timestamp":"2022-12-10T11:28:03.310+00:00","status":400,"error":"Bad Request","path":"/actuator/metrics/metricWithCommaInTag"}```

**Expected:**
metric is displayed in Spring Boot Admin.

**Why:**
Because SBA concats all tags into single one `?tag=<key:value>,<key:value>` and Spring will split this string by comma (it is incorrect if the comma is inside tag)

**How should be done**
Should use following syntax: `?tag=<key:value>&tag=<key:value>`

### CODE REVIEW

File: spring-boot-admin-server-ui/src/main/frontend/services/instance.js [hunk 1]
```diff
@@ -67,12 +67,24 @@ class Instance {
...
+      Object.entries(tags)
         .filter(([, value]) => typeof value !== 'undefined' && value !== null)
-        .map(([name, value]) => `${name}:${value}`)
-        .join(',')
-    } : {};
+        .forEach(([name, value]) => {
+          params.append('tag', `${name}:${value}`);
+
+          if (!firstElementDuplicated) {
+            // workaround for tags that contains comma
```
Comment (erikpetzold) [inline spring-boot-admin-server-ui/src/main/frontend/services/instance.js:79]: Do we really need to always duplicate the first param? Would be only needed in very rare case of having only 1 param and the param is containing a comma, right?

Comment (Fuud) [reply spring-boot-admin-server-ui/src/main/frontend/services/instance.js:79]: Yes. I just wrote a simple solution. As duplicate does not break anything.
Do you want me to rewrite code with more precice condition?

Comment (erikpetzold) [reply spring-boot-admin-server-ui/src/main/frontend/services/instance.js:79]: is ok

**Comment by erikpetzold** [comment]
@Fuud change is included in 2.7.10, released just now

**Pull request merged**